docs/docs/adding-new-aws-iam-user.md

```diff
@@ -0,0 +1,34 @@
+# Adding a New AWS IAM User
+
+We manage our AWS infrastructure in Terraform. Creating a new user with console access in Terraform requires a signing key to encrypt the password that will be generated for the user. A signing key is stored in `infra/public.pgp`.
+
+Although you should not need to generate a new key, in case you do (e.g., if the original key becomes compromised), follow these steps:
+
+1. Start a shell in the `infra` docker container, binding the directory to `/infra`: `docker run -it -v $(pwd):/infra infra /bin/sh`
+1. In the container's shell, generate a new key: `gpg --batch --passphrase '' --quick-gen-key spelmanaws@spelman.edu default default`
+1. Export the public key to a file which Terraform will use: `gpg --export --armor | head -n -2 | tail +3 > public.pgp`. The `head` and `tail` commands remove the header, checksum, and footer since Terraform does not expect these as part of the file.
```

Comment:
I think `--export` defaults to exporting all keys, which is I'm thinking is OK, since these are public keys. I believe we can supply the email as the argument to the `--export` command for just that pub key.

Reply:
Good to know. Since we're doing this in the container, the keystore is ephemeral anyways. Once you exit your session in the image it gets deleted. So it doesn't matter here.

But I'm open to adding the email to the `--export` command if you think it's a good idea.

Reply:
No worries! Thanks!